datarepo-clienttests/src/main/java/common/CommandCLI.java

```diff
@@ -92,7 +92,10 @@ public static void printHelp() throws IOException {
 
   public static void runTestMain(String[] args) throws Exception {
     if (args.length == 2) { // execute a test configuration or suite
-      TestRunner.executeTestConfigurationOrSuite(args[0], args[1]);
+      boolean isFailure = TestRunner.executeTestConfigurationOrSuite(args[0], args[1]);
+      if (isFailure) {
+        System.exit(1);
```

Comment:
Do you need to explicitly return 0 on success as well?

Reply:
If no one has called System.exit explicitly, Java returns an exit code of 0 once all threads terminate, so there's no need to return 0 explicitly.